templates/spec/system/products_spec.rb

```diff
@@ -166,7 +166,8 @@
 
     it 'displays price of first variant listed', js: true do
       click_link product.name
-      within("[data-js='price']") do
+
+      within("#product-price") do
```

Comment:
I wasn't able to figure out why the "Copy product form from Solidus Demo" commit caused this spec example to fail.